app/components/DetailsDropdownButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Icon } from '@trussworks/react-uswds'
+import type { ButtonProps } from '@trussworks/react-uswds/lib/components/Button/Button'
+import type { ReactNode } from 'react'
+
+export default function DetailsDropdownButton({
+  children,
+  className,
+  ...props
+}: {
+  children: ReactNode
+  className?: string
+} & Omit<ButtonProps & JSX.IntrinsicElements['button'], 'type'>) {
+  return (
+    <Button
+      className={`usa-button ${className ?? ''}`}
```

Comment:
Isn't `usa-button` already applied by the `<Button>` component?

Reply:
Yup, that was another accidental holdover from the details/summary version. Fixing now